src/public/drug/WithdrawnNotice.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Card from '@material-ui/core/Card';
+import CardHeader from '@material-ui/core/CardHeader';
+import CardContent from '@material-ui/core/CardContent';
+import WarningIcon from '@material-ui/icons/Warning';
+import Typography from '@material-ui/core/Typography';
+import withStyles from '@material-ui/core/styles/withStyles';
+
+const styles = theme => {
+  return {
+    root: {
+      marginTop: 8,
+      border: `2px solid ${theme.palette.secondary.main}`,
+    },
+    header: {
+      padding: '8px 16px',
+      color: 'white',
+      backgroundColor: theme.palette.secondary.main,
+    },
+    content: {
+      padding: '8px 16px !important',
```

Comment:
I'm guessing the `!important` was necessary?

Reply:
Removed.